src/Controls/tests/TestCases.HostApp/Issues/Issue25473.xaml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+				 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+				 xmlns:local="clr-namespace:Maui.Controls.Sample.Issues"
+				 x:Class="Maui.Controls.Sample.Issues.Issue25473">
+
+  <VerticalStackLayout>
```

Comment:
LGTM but, could the test include two Entries, using the two https://learn.microsoft.com/es-es/dotnet/api/microsoft.maui.clearbuttonvisibility?view=net-maui-8.0 enum values?

Will require to validate each one with a snapshot but in this way we have a test verifying all the ClearButtonVisibility possibility.

Reply:
@jsuarezruiz ,Thank you for the feedback! Based on your suggestion, I have added the additional test cases to include both ClearButtonVisibility enum values. Each entry is now validated with a snapshot, ensuring we have comprehensive coverage for all ClearButtonVisibility scenarios. Please let me know if there are any further adjustments needed.